dot_lpdev_cmds.sh

```diff
@@ -485,15 +450,12 @@ function __lpdev_protocol_deploy {
 
   OPWD=$PWD
   cd $srcDir/protocol
-  migrateCmd="truffle migrate --reset --network lpTestNet"
+  migrateCmd="npm run migrate -- --network=lpTestNet"
```

Comment:
why do we need the `--` that is between `migrate` and `--network` ?

Reply:
Fixed in a27d94d